src/components/Contacts/ContactPartnershipStatus/ContactLateStatusLabel/ContactLateStatusLabel.test.tsx

```diff
@@ -1,54 +1,121 @@
 import React from 'react';
 import { ThemeProvider } from '@mui/material/styles';
 import { render } from '@testing-library/react';
+import { Settings } from 'luxon';
 import theme from '../../../../theme';
-import {
-  ContactLateStatusEnum,
-  ContactLateStatusLabel,
-} from './ContactLateStatusLabel';
+import { ContactLateStatusLabel } from './ContactLateStatusLabel';
+
+Settings.now = () => new Date(2020, 1, 1).valueOf();
 
 describe('ContactLateStatusLabel', () => {
-  it('should render on time', () => {
+  it('should render on time when both dates are in the future', () => {
     const { getByText } = render(
       <ThemeProvider theme={theme}>
-        <ContactLateStatusLabel lateStatusEnum={ContactLateStatusEnum.OnTime} />
+        <ContactLateStatusLabel
+          lateAt="2020-02-06T12:00:00Z"
+          pledgeStartDate="2020-02-04T12:00:00Z"
+        />
       </ThemeProvider>,
     );
 
     expect(getByText('(On time)')).toBeInTheDocument();
   });
 
-  it('should render less than 30 days late', () => {
+  // this case should never happen but it is worth testing
```

Comment:
Are you sure this can never happen? If the donations are on time, wouldn't `lateAt` be in the future?